faasmtools/build.py

```diff
@@ -71,7 +71,9 @@
 # introduced them upstream. Use of atomics means we can't link things together:
 # https://reviews.llvm.org/D59281
 WASM_CFLAGS = [
-    "-O3 -mno-atomics",
+    "-O3",
+    "-mno-atomics",
+    "-mno-simd128",
```

Comment:
Note that in LLVM 13 we need to be explicit about `-mno-simd128` as otherwise some targets are built with `simd`.